.github/workflows/deploy.yml

```
@@ -1,121 +1,101 @@
+        with:
+          distribution: "temurin"
+          java-version: "21"
+          cache: maven
+
+      - name: Build with Maven
+        run: |
+          cd goweb-spring
+          mvn clean package -DskipTests
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)